modules/contacts/client/components/ContactList.component.js

```diff
@@ -19,18 +21,16 @@ export default class ContactList extends React.Component {
   }
 
   render() {
-
     const { appUser, contacts, onContactRemoved } = this.props;
+    const { t } = useTranslation('contact');
```

Comment:
https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both

> You can’t use Hooks inside of a class component

Reply:
Ahh! Good catch — I'm still new to these.

Reply:
Addressed in https://github.com/Trustroots/trustroots/pull/1162/commits/9c8ba0193b74aea0e4b05be0b080b3d4fa75fdbf